.github/workflows/release.yaml

```diff
@@ -0,0 +1,42 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+
+  pull_request:
+
+env:
+  MAIN_REPO: IN-CORE/pyincore-data
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: get release info
+        id: release_info
+        run: |
+          version=$(awk -F= '/^release/ { print $2}' docs/source/conf.py | sed "s/[ ']//g")
+          changelog="$(sed -e "1,/^## \[${version}\]/d" -e "/^## \[/,\$d" -e '/^$/d' CHANGELOG.md)"
+          changelog="${changelog//'%'/'%25'}"
+          changelog="${changelog//$'\n'/'%0A'}"
+          changelog="${changelog//$'\r'/'%0D'}"
+          echo "::set-output name=version::$version"
+          echo "::set-output name=changelog::$changelog"
+
+      - name: create release
+        if: github.event_name != 'pull_request' && github.repository == env.MAIN_REPO
+        uses: actions/create-release@v1
```

Comment:
I'm not sure if we should be concerned about this.. `actions/create-release` is not being maintained anymore. In the near future, should we try to move to one of the actions that are active? Some alternatives are listed here: https://github.com/actions/create-release

Reply:
nice catch, switched to https://github.com/softprops/action-gh-release